custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -467,7 +474,7 @@ def _prepare_fit_pipeline(self, run_language):
             else:
                 raise DrumCommonException(
                     "Target type {} requires class label information. No labels were supplied and "
-                    "labels could not be inferred from the target."
+                    "labels could not be inferred from the target.".format(self.target_type.value)
```

Comment:
unrelated, but uncovered while working on this and figured we may as well tell people what target type we think they're using if they hit this error